requirements.txt

```diff
@@ -9,3 +9,4 @@ pypandoc
 pytest-benchmark
 pytest-docker-pexpect
 twine
+openai
```

Comment:
The introduction of a new hard dependency has huge consequences downstream. Not to mention this is a dependency for something optional.